build.gradle

```diff
@@ -48,3 +48,5 @@ subprojects {
 task wrapper(type: Wrapper) {
     gradleVersion = '4.0.1'
 }
+
+defaultTasks 'build'
```

Comment:
Was this change intentionally checked in?

Reply:
Yup.  It will make Gradle do the thing which most devs will probably expect.  Do you think it's bad?

Reply:
I wasn't familiar with that feature.

I think `'clean', 'build'` might be a better default

Reply:
Well, in a regular dev cycle, you will want to call `build` many times, but `clean` only rarely.  I'd rather not pay the cost of `clean` every time, and having to specify `build` every time defeats the purpose of making it the default!

Reply:
Sounds good